ansible_collections/arista/avd/roles/dhcp_provisioner/templates/ztp_configuration.j2

```diff
@@ -56,6 +58,8 @@ ztp:
         end: {{ ztp_pool_end }}
 {% endif %}
 {% if ztp_lease_time is arista.avd.defined %}
+        lease_time: {{ ztp_lease_time }}
+{% else %}
         lease_time: 300
```

Comment:
I think we can remove this else condition. As ztp_lease_time is set to 300 in the defaults/main.yml of this role. So by default it is set to 300 unless if user defines it. 

So we can change lines 61-64 to ```lease_time: {{ ztp_lease_time }}```

Reply:
I have removed `if/else` as default value for `ztp_lease_time` was already configured